plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -118,7 +118,7 @@
 		  scope="total"
 		  class="com.google.cloud.tools.eclipse.appengine.validation.AppEngineWebXmlSourceValidator"
 		  id="appEngineWebXmlSourceValidator">
-		<contentTypeIdentifier id="appengine-web.xml">
+		<contentTypeIdentifier id="com.google.appengine.dd.xml">
```

Comment:
This is validating appengine-web.xml. I changed it based on Brian's suggestion. Should probably have been in a separate PR.